pages/docs/comparison.mdx

```diff
@@ -91,7 +91,26 @@ author your own light and dark mode experience across your application.
 
 ## How is Chakra different from Material UI?
 
-// TO DO (Feel free to submit a PR)
+The Material UI React library provides users with a handful of UI layout tools 
+but is most famous for its large breadth of pre-styled UI components to which 
+developers can apply custom styles to override out-of-the-box base styles. Chakra 
+UI is a more robust, layout-focused library that also provides developers with UI 
+components similar to those that Material UI provides, but has a greater focus on 
+the creation of flexible, composable, and scalable code. 
+
+A core concept to be considered when comparing the two frameworks is 
+'Ease of Modification'. Comparatively, Material UI adds far more classes to 
+individual HTML tags related to exported components and layouts, forcing developers 
+to fight against base styles when customizing their interface. Chakra gives developers 
+far more freedom to manipulate CSS classes of exported components and layouts and 
+often requires less code to do so. For example, Material UI requires separate code to 
+control responsive styling based on viewport changes, while Chakra provides built-in 
+support for responsive styling without the need to create CSS classes or media queries. 
```

Comment:
@lazarnikolov94 @TuckerMassad 
I would say this is not true anymore with MUI v5 https://mui.com/system/basics/#responsive-values

Reply:
Chakra UI might benefit from clarifying this part. I imagine it would give more credibility to the rest of the description in this section. Here is my flow of thought, a developer familiar with MUI might pause here with: "Wait, this is not true. Can I trust what I have read until now? Maybe not".